fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -188,7 +191,10 @@ def fit(self, X, y, *, sensitive_features, **kwargs):
         else:
             raise ValueError(NOT_SUPPORTED_CONSTRAINTS_ERROR_MESSAGE)
 
-        _, _, sensitive_feature_vector = _validate_and_reformat_input(
+        if kwargs.get(_KW_CONTROL_FEATURES) is not None:
+            raise ValueError(NO_CONTROL_FEATURES)
```

Comment:
Is this something we want at some point? If so, let's briefly describe this and put it in an issue with "help-wanted"? It should be simple enough I think. You'd just create individual objects per control group.

Reply:
I can certainly file an issue for it.